Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -1809,6 +1810,34 @@ def com_google_fonts_check_metadata_subsets_order(family_metadata):
     yield PASS, "METADATA.pb subsets are sorted in alphabetical order."
 
 
+@check(
+  id = 'com.google.fonts/check/metadata/includes_production_subsets',
+  conditions = ['family_metadata', 'production_metadata', 'listed_on_gfonts_api'],
```

Comment:
regarding code-style, I prefer having one condition per line (aligned at the opening single-quote)